src/compiler/commandLineParser.ts

```diff
@@ -2484,13 +2501,25 @@ namespace ts {
 
         const options = getDefaultTypeAcquisition(configFileName);
         const typeAcquisition = convertEnableAutoDiscoveryToEnable(jsonOptions);
-        convertOptionsFromJson(typeAcquisitionDeclarations, typeAcquisition, basePath, options, Diagnostics.Unknown_type_acquisition_option_0, errors);
+
+        const diagnostics = {

```

Comment:
This seems incorrect? It needs unknown type aquisition? and corresponding did you mean

Reply:
I feel like this should be right:

```diff
From 

- Diagnostics.Unknown_type_acquisition_option_0, errors

to

+ unknownOptionDiagnostic: Diagnostics.Unknown_type_acquisition_option_0
+ unknownDidYouMeanDiagnostic: Diagnostics.Unknown_type_acquisition_option_0_Did_you_mean_1
```